ios/src/TGGeoPolygon.mm

```diff
@@ -0,0 +1,80 @@
+//
+//  TGGeoPolygon.mm
+//  tangram
+//
+//  Created by Karim Naaji on 10/27/16.
+//
+//
+
+#import "TGGeoPolygon.h"
+
+#include <vector>
+
+@interface TGGeoPolygon () {
+    std::vector<TGGeoPoint> coordinates;
+    std::vector<int> rings;
```

Comment:
Convert them to NSNumbers and use NSMutableArray if you want to stay purely within the scope of foundation classes. There are several shorthand ways of generating NSNumbers, like `NSNumber *num = @9;`. So you could do something like `NSMutableArray *array = @[@9, @10];` 

That said, a major selling point of Objective-C is its interoperability with C and C++, so you don't have to confine yourself like that.